### PR TITLE
Update http remote documentation

### PR DESCRIPTION
**Pull request by MetalBlueberry**
documentation for #4547 and #4517

### CODE REVIEW

**Comment by MetalBlueberry** [comment]
Editing from the GitHub UI is hard `¯\_(ツ)_/¯` 

I think this should be a squash merge to remote all the unnecessary commits.

**Comment by jorgeorpinel** [comment]
Per
https://github.com/iterative/dvc/pull/4517
and
https://github.com/iterative/dvc/pull/4547

File: content/docs/command-reference/remote/modify.md [hunk 1]
```diff
@@ -572,6 +572,8 @@ more information.
   $ dvc remote modify myremote url https://example.com/path/to/dir
   ```
 
+  > The url can include query parameters that will be preserved.
```
Comment (jorgeorpinel) [inline content/docs/command-reference/remote/modify.md:575]:
```suggestion
  > The URL can include a query string, which will be preserved (e.g.
  > `example.com?loc=path%2Fto%2Fdir`)
```

File: content/docs/command-reference/remote/modify.md [hunk 2]
```diff
@@ -630,6 +633,12 @@ more information.
   > `password` is specified, DVC will not prompt the user to enter a password
   > for this remote.
 
+- `ssl_verify` - allows to disable ssl verification. It is enabled by default.
```
Comment (jorgeorpinel) [inline content/docs/command-reference/remote/modify.md:636]:
```suggestion
- `ssl_verify` - allows to disable SSH verification, which is enabled by default.
```

**Review by jorgeorpinel — CHANGES_REQUESTED** [review]
Thanks @MetalBlueberry !

**Review by jorgeorpinel — APPROVED** [review]
Oops, meant to approve.

**Comment by jorgeorpinel** [comment]
Something is wrong with Restyled @shcheklein ☝️ It's not showing up 😕 — let's see if it's a temporary outage.

I'll fix the formatting manually for this one. Let's keep an eye on that.

File: content/docs/command-reference/remote/modify.md [hunk 3]
```diff
@@ -630,6 +633,12 @@ more information.
   > `password` is specified, DVC will not prompt the user to enter a password
   > for this remote.
 
+- `ssl_verify` - allows to disable SSH verification, which is enabled by default.
```
Comment (jorgeorpinel) [inline content/docs/command-reference/remote/modify.md:636]:
```suggestion
- `ssl_verify` - allows to disable SSH verification, which is enabled by
  default.
```

**Pull request merged**